components/infobox/wikis/rainbowsix/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,248 @@
+						return OperatorIcon.getImage{operator, size = _SIZE_OPERATOR}
+					end
+				)
+				return {
+					Cell{
+						name = #operatorIcons > 1 and 'Signature Operators' or 'Signature Operator',
+						content = {
+							table.concat(operatorIcons, '&nbsp;')
+						}
+					}
+				}
+			end
+		})
+	-- Active in Games
+	table.insert(widgets,
+		Builder{
+			builder = function()
+				local activeInGames = {}
+				Table.iter.forEachPair(_GAMES,
+					function(key)
+						if _args[key] then
+							table.insert(activeInGames, _GAMES[key])
+						end
+					end
+				)
+				return {
+					Cell{
+						name = #activeInGames > 1 and 'Games' or 'Game',
+						content = {
+							table.concat(activeInGames, '<br>')
+						}
+					}
+				}
+			end
+		})
+	return widgets
+end
+
+function CustomPlayer:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomPlayer:adjustLPDB(lpdbData)
+	lpdbData.extradata.role = Variables.varDefault('role')
+	lpdbData.extradata.role2 = Variables.varDefault('role2')
+
+	lpdbData.extradata.signatureOperator1 = _args.operator1 or _args.operator
+	lpdbData.extradata.signatureOperator2 = _args.operator2
+	lpdbData.extradata.signatureOperator3 = _args.operator3
+	lpdbData.extradata.signatureOperator4 = _args.operator4
+	lpdbData.extradata.signatureOperator5 = _args.operator5
+	lpdbData.extradata.isplayer = Variables.varDefault('isplayer')
+	lpdbData.type = Variables.varDefault('isplayer') == 'true' and 'player' or 'staff'
+
+	lpdbData.region = Template.safeExpand(mw.getCurrentFrame(), 'Player region', {_args.country})
+
+	if String.isNotEmpty(_args.team2) then
+		lpdbData.extradata.team2 = Team.page(_args.team2)
+	end
+
+	return lpdbData
+end
+
+function CustomPlayer:createBottomContent(infobox)
+	if Namespace.isMain() and String.isNotEmpty(_args.team) then
+		local teamPage = Team.page(mw.getCurrentFrame(),_args.team)
+		return
+			Template.safeExpand(mw.getCurrentFrame(), 'Upcoming and ongoing matches of', {team = teamPage}) ..
+			Template.safeExpand(mw.getCurrentFrame(), 'Upcoming and ongoing tournaments of', {team = teamPage})
+	end
+end
+
+function CustomPlayer._getStatusContents()
+	local statusContents = {}
+
+	if not String.isEmpty(_args.status) then
+		table.insert(statusContents, Page.makeInternalLink({onlyIfExists = true}, _args.status) or _args.status)
+	end
+
+	local banned = _BANNED[string.lower(_args.banned or '')]
+	if not banned and not String.isEmpty(_args.banned) then
+		banned = '[[Banned Players|Multiple Bans]]'
+		table.insert(statusContents, banned)
+	end
+
+	statusContents = Array.extendWith(statusContents,
+		Array.map(Player:getAllArgsForBase(_args, 'banned'),
+			function(item, _)
+				return _BANNED[string.lower(item)]
+			end
+		)
+	)
+
+	return statusContents
+end
+
+function CustomPlayer._createRole(key, role)
+	if String.isEmpty(role) then
+		return nil
+	end
+
+	local roleData = _ROLES[role:lower()]
+	if not roleData then
+		return nil
+	end
+	if Namespace.isMain() then
```

Comment:
maybe use `Player:shouldStoreData` instead of `Namespace.isMain()`
(same maybe on the other use cases)